src/components/TokensCard/TokensCard.tsx

```diff
@@ -79,6 +77,31 @@ export const TokensCard: FC<TTokensCard> = ({ collectionId, tokenId, offer, test
     return {};
   }, [collectionId, tokenId, collection, token, api]);
 
+  useEffect(() => {
```

Comment:
utils

seems overcomplicated for effect

Reply:
fixed